litmus-portal/graphql-server/pkg/handlers/readiness_handler.go

```diff
@@ -27,39 +27,40 @@ func contains(s []string, str string) bool {
 	return false
 }
 
-func ReadinessHandler(handler http.Handler, mclient *mongo.Client) http.Handler {
-	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+// ReadinessHandler returns readiness information
+func ReadinessHandler(mclient *mongo.Client) gin.HandlerFunc {
+	return func(c *gin.Context) {
 		var (
-			db_flag  = "up"
-			col_flag = "up"
+			dbFlag  = "up"
+			colFlag = "up"
 		)
 
 		dbs, err := mongodb.Operator.ListDataBase(context.Background(), mclient)
 		if err != nil {
-			db_flag = "down"
+			dbFlag = "down"
```

Comment:
please print to error. Useful for debugging.

```suggestion
			logrus.Error(err)
```